[EN/RU] Update translations of "About", "osu!api" and "Rules"; fix a couple of small issues in "Rules" (#2641)
ppy/osu-wiki: wiki/Rules/en.md

1. **No topic in Off-Topic has value.** They are put simply, trash. If your threads keep getting moved to Off-Topic, they are being judged to be unworthy of serious discussion. Take the hint.

> 2. **Excessive stupidity is bad.** Being endearingly dumb is fine, and falls into the modus operandi of shitposting. Genuine stupidity is never fine. If you are unable to distinguish the difference, you're already breaking the rule.

Reviewer: hey, this is still a rule of off-topic!

honestly why are these even listed here lol

These bad things may include (*but are not limited to*):

* The removal of your ability to chat in-game for a period of time (known as a silence).
* The removal of any infringing content from your userpage, profile or uploaded beatmaps.
* The loss of the privilege to edit or alter your userpage or profile details.
* The loss of the privilege to play or qualify in officially hosted tournaments.

Silences and other infringements **automatically** increase in duration near-exponentially with each infringement. Moderators do not control the length of time you are silenced for an offense - repeat offenders will find themselves locked out of the game for increasingly longer periods of time.

Certain offenses carry a minimum duration (usually 48 hours or more) if they are particularly severe, though they are almost exclusively reserved for severe infractions, such as sharing graphic content or unrepentant harassment of other users.

If after several infringements you continue to break the rules, your account may be permanently and irrevocably restricted.

> Restricted accounts can not play online, set scores, chat with other users and are for all intents and purposes 'banned' from interaction with the community. Restriction for community rule violation can only be removed upon appeal to the game staff after 3 months have passed ([conditions may apply](/wiki/Help_Centre/Account_Restrictions#common-restriction-reasons-and-cooldowns)).

Reviewer: ```suggestion
Restricted accounts cannot play online, set scores, chat with other users and are for all intents and purposes "banned" from interaction with the community. Restriction for community rule violation can only be removed upon appeal to the support staff after 3 months have passed ([more conditions may apply](/wiki/Help_Centre/Account_Restrictions#common-restriction-reasons-and-cooldowns)).
```

Author: applied this in my commit along with other corrections

Reviewer: missed some parts of it 👀 try using the "commit suggestion" button if u want to apply it directly next time, that way errors don't happen and I don't have to check again 👍
